lib/util/rule-fixer.js

```diff
@@ -34,14 +34,9 @@ function insertTextAt(index, text) {
 
 /**
  * Creates code fixing commands for rules.
- * @constructor
  */
-function RuleFixer() {
-    Object.freeze(this);
-}
 
-RuleFixer.prototype = {
-    constructor: RuleFixer,
+const ruleFixer = {
```

Comment:
We want to freeze this object so that we are consistent with what we had previously.